www/install/php/Update-22.10.0-beta.1.php

```diff
@@ -35,6 +35,10 @@
 
     $errorMessage = "Unable to delete 'appKey' information from database";
     $pearDB->query("DELETE FROM `informations` WHERE `key` = 'appKey'");
+    if ($pearDB->isColumnExist('remote_servers', 'app_key') === 1) {
+        $errorMessage = "Unable to drop 'app_key' from remote_servers table";
+        $pearDB->query("ALTER TABLE remote_servers DROP COLUMN `app_key`");
+    }
 
     $pearDB->commit();
```

Comment:
must be after commit, because `ALTER` triggers an autocommit
```suggestion
    $pearDB->commit();
    
    if ($pearDB->isColumnExist('remote_servers', 'app_key') === 1) {
        $errorMessage = "Unable to drop 'app_key' from remote_servers table";
        $pearDB->query("ALTER TABLE remote_servers DROP COLUMN `app_key`");
    }
```

Reply:
@kduret done :D !